assets/dev/scss/frontend/_global.scss

```diff
@@ -124,6 +124,17 @@
 	}
 }
 
+.elementor {
+
+	:is( .e-container ) {
+
+		// Hack to increase specificity and avoid using `!important`.
+		& > .elementor-widget.elementor-widget {
```

Comment:
I am not sure if I like this. What happens in section/column? Why does it only happen in container?
Is anything different if you set the container to `flex-wrap: wrap`?

Reply:
@udidol 

Inside section/column: a max-width value has been set here: which is equal to the width. I really don't know why. I didn't want to change anything for the section/column.
https://github.com/marcelementor/elementor/blob/2fba924811524d429f1a83a8354bcc95c325abe6/includes/widgets/common.php#L259. 

It only happens when using `flex column`. Using `flex-wrap: wrap` doesn't make a difference.